packages/postgres-database/src/simcore_postgres_database/utils_payments.py

```diff
@@ -0,0 +1,152 @@
+import datetime
+from dataclasses import dataclass
+from decimal import Decimal
+from typing import TypeAlias
+
+import sqlalchemy as sa
+from aiopg.sa.connection import SAConnection
+from aiopg.sa.result import ResultProxy, RowProxy
+
+from . import errors
+from .models.payments_transactions import PaymentTransactionState, payments_transactions
+
+PaymentID: TypeAlias = str
+
+PaymentTransactionRow: TypeAlias = RowProxy
+
+
+@dataclass
+class PaymentFailure:
```

Comment:
I would have expected this to be a pedantic model.

Reply:
`pydantic` you mean :-)
smcor_postgres_database ha no pydantic library installed